### DISCUSSION TITLE
ETA of new release will fixing vulnerabilities issues

### DISCUSSION THREAD

**Discussion by EzequielAlejandroLastra** [JLLeitschuh/ktlint-gradle#808] [General]
Hi,

Do you have an ETA for the next release ? I need to fix some vulnerabilities issues in my services and I see some dependencies are used in this plugin. For example: CVE-2023-6378

Anyway thanks for the great work on this plugin 😃

**Comment by JLLeitschuh** [comment]
Done! Thanks! https://github.com/JLLeitschuh/ktlint-gradle/releases/tag/v12.1.2
